1-js/04-object-basics/06-constructor-new/article.md

```diff
@@ -185,7 +185,7 @@ Omitting parentheses here is not considered a "good style", but the syntax is pe
 
 Using constructor functions to create objects gives a great deal of flexibility. The constructor function may have parameters that define how to construct the object, and what to put in it.
 
-Of course, we can add to `this` not only properties, but methods as well.
+Of course, we can add `this` to, not only properties, but methods as well.
```

Comment:
but we add the stuff "to this"
just asking, not native speaker 

```suggestion
Of course, we can add not only properties, but also methods to `this`.
```

Reply:
Yes the edit you made would work fine as well.

Reply:
Fixed grammatical typo